src/m365/aad/commands/user/user-set.ts

```diff
@@ -81,18 +121,41 @@ class AadUserSetCommand extends GraphCommand {
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     try {
+      if (this.verbose) {
+        logger.logToStderr(`Updating user ${args.options.userPrincipalName || args.options.objectId}`);
+      }
+
+      if (args.options.currentPassword) {
+        if (args.options.objectId && args.options.objectId !== accessToken.getUserIdFromAccessToken(auth.service.accessTokens[auth.defaultResource].accessToken)) {
+          throw `Please specify your own objectId when wishing to change your password`;
```

Comment:
Let's reword it to make it easier to understand: 'You can only change your own password. Please use `--objectId @meId` to reference to your own userId'